addons/l10n_fr_invoice_addr/views/account_move_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_move_form_l10n_fr_invoice_addr" model="ir.ui.view">
+        <field name="name">l10n_fr_invoice_addr.account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_shipping_id']" position="before">
+                <field name="l10n_fr_is_company_french" invisible="1"/>
+            </xpath>
+            <xpath expr="//field[@name='partner_shipping_id']" position="attributes">
+                <attribute name="groups"/>
+                <attribute name="attrs">
+                    {'invisible': [
+                        ('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt')),
+                        ('l10n_fr_is_company_french', '=', False),
```

Comment:
I don't get why we want to show that for vendor bills/refunds/misc entries for french companies...

How does that relate to the law?